config/jobs/kubernetes-sigs/jobset/jobset-presubmit.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
```

Comment:
please consider adding an OWNERS file scoped to kubernetes-sigs/jobset so jobset maintainers can self approve small changes to this.

Reply:
@BenTheElder just added an OWNERS file which accidentally removed your LGTM, sorry about that

Reply:
NBD, lgtm anyone in the org can add as well :-)

Reply:
Thank you!